README.md

```diff
@@ -9,52 +9,53 @@ We decided to build upon this work and build and RDF based ontology, for the *DF
 
 ![](./docs/dfgfo-hierarchies.png)
 
+## Ontology
 
-
-## Ontology 
 * **Ontology TTL**: [dfgfo.ttl](./dfgfo.ttl)
-* **Ontology IRI**: https://github.com/tibonto/dfgfo/
+* **Ontology IRI**: <https://github.com/tibonto/dfgfo/>
 * **Ontology PURL**: <https://raw.githubusercontent.com/tibonto/DFG-Fachsystematik-Ontology/main/dfgfo.ttl>
 * **ontology prefix/id**: `dfgfo`
 
-
-## Create/update ontology 
+## Create/update ontology
 
 **[dfgfo.ttl](./dfgfo.ttl) ontology file is created, by [scripts/create_ontology.py](./scripts/create_ontology.py) python script**, which
-* parses the DFG classification system encoded [csv/Fachsystematik_2020-2024.csv](./csv/Fachsystematik_2020-2024.csv) (in EN/DE)
+
+* parses the DFG classification system encoded in csv/Fachsystematik_20XX-20XX.csv (in EN/DE) (cf. directory [csv/](/csv/) and [csv/README.md](/csv/README.md))
 * encodes each of the DFG's classification subjects (in .csv cells) into RDF graph triples
-    * of type `owl:Class`
-    * with `rdfs:label` in EN and skos:altLabel in DE
-    * subsumed to parent subject with `rdfs:subClassOf` accordinng to DFG Classification hierarchy 
+  * of type `owl:Class`
+  * with `rdfs:label` in EN and skos:altLabel in DE
+  * subsumed to parent subject with `rdfs:subClassOf` accordinng to DFG Classification hierarchy
 * parses the metadata triples from [metadata.ttl](./metadata.ttl) into a graph
 * joins metadata and DFG classification graphs into [dfgfo.ttl](./dfgfo.ttl)
 
-
-**Run**
+### Run
 
 Create a python3 Virtual Environment
 
 Install requirements `pip install -r scripts/requirements.txt`
 
 Run script to create ontology `python scripts/create_ontology.py`. Make sure to use end of line sequence `LF` for  [/csv/Fachsystematik_2020-2024.csv](/csv/Fachsystematik_2020-2024.csv).
```

Comment:
@SArndt-TIB I am not sure if the following sentence is understandable. I dont understand it myself

> Make sure to use end of line sequence `LF` for [/csv/Fachsystematik_2020-2024.csv](/csv/Fachsystematik_2020-2024.csv).

Shall we do something about it? remove it??

Reply:
@andrecastro0o I am working with VS Code on Windows and accidentally introduced a lot of line breaks once, since I did not switch this when handling the files. It is possible to choose the end of line sequence in VS Code.  Switching from CR LF to LF solved the problem of introducing line breaks in the ontology. I tried to make people aware without going into too much detail. See also https://stackoverflow.com/questions/1552749/difference-between-cr-lf-lf-and-cr-line-break-types - maybe more context is needed on that one?